RecipesAppTeamPlate/RecipesAppTeamPlate/ProfileModule/Model/InfoSource.swift

```diff
@@ -0,0 +1,25 @@
+// InfoSource.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+final class InfoSource: InfoSourceProtocol {
```

Comment:
```suggestion
final class InfoSource: InfoSourceProtocol {
```
Source общее название